test/streamparse/test_streamparse_run.py

```diff
@@ -0,0 +1,42 @@
+"""
+Tests for streamparse_run
+"""
+
+import logging
+import os
+import sys
+import unittest
+
+from streamparse import run
+
+
+class StreamparseRunTests(unittest.TestCase):
+
+    run_target_invoked = False
+    run_target_invoked_serializer = None
+
+    def test_streamparse_run_storm_1_0_2(self):
+        # patch streamparse_run sys argv assuming tests run
+        # from streamparse project root, (Storm <= 1.0.2 case)
+        sys.argv = ['streamparse_run',
+                    'test.'
```

Comment:
It is not an extra argument, (no comma following the 'test.' string). The two strings are appended into a long string. Long string was split to place emphasis on the added 'test.' path required to reach the target bolt, (simulates the extra 'resources' folder in path for Storm <= 1.0.2). There is another split on the 2nd test case below. There the --serializer option is concatenated to the argument.

Let me know if you want me to make this more clear by pulling these together into one string. :-)